src/main/docs/guide/quickStart/usingsnapshots.adoc

```diff
@@ -0,0 +1,24 @@
+Micronaut milestone and stable releases are distributed to https://bintray.com/micronaut[Bintray].
+
+The following snippet shows how to use Micronaut `BUILD-SNAPSHOT` with Gradle:
+
+[source, groovy]
+----
+ext {
+  micronautVersion = '1.0.0.BUILD-SNAPSHOT'
+}
+repositories {
+  mavenLocal()
+  mavenCentral()
+  jcenter() <1>
+  maven { url "https://oss.sonatype.org/content/repositories/snapshots/" } // <2>
+}
+dependencyManagement {
+  imports {
+      mavenBom "io.micronaut:bom:$micronautVersion"
```

Comment:
The default generated `build.gradle` that comes out of `mn create-app <app name>` doesn't define `micronautVersion`.  I think the only place the version number is mentioned in the project is directly in the box import statement...

```groovy
dependencyManagement {
    imports {
        mavenBom 'io.micronaut:bom:1.0.0.M3'
    }
}
```

If we are going to advocate for the approach shown above for using snapshots, I think we should make the default generated build file use the same approach.  Instead of `ext` I think defining it in `gradle.properties` might be better, but the main thing would be to be consistent.  Instead of having to define new things in order to use a `BUILD-SNAPSHOT`, I think the developer ought to just be able to change a number in 1 place.  As it is right now, that one place would be in the bom import.  

All of that of course is separate from needing to add the snapshot maven dependency repo, which of course needs to be done either way.

Reply:
I prefer defining version in `gradle.properties` instead of ext